### PR TITLE
Supporting the 52Fund

### PR DESCRIPTION
**Pull request by ross-spencer**
Connected to #20

### CODE REVIEW

File: _posts/2020-01-26-support-the-52Fund.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+organizations that have leadership in their mandate and are failing to
+demonstrate that. That is what we mean by leadership by nature. Processes and
+procedures can often help us but this time they are failing a group of good
+people.
+
+The SAA community, and existing leadership (by virtue of _position_, or
+_**nature**_, have spoken and are looking to support Lae’l Hughes-Watkins
+[call][saa-7].
+
+They have setup a fund for underrepresented communities to access to have
```
Comment (andreakb) [inline _posts/2020-01-26-support-the-52Fund.md]:
```suggestion
They have setup the *[#52Fund][saa-8]* to support QTBIPOC, BIPOC, LGBTQ+, and disabled memory workers' participation in official professional channels, like renewing SAA memberships and attending SAA's annual meeting.
```

File: _posts/2020-01-26-support-the-52Fund.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+their voice on this situation as well and that is the *[#52Fund][saa-8]*.
+
+Please read about the mission, and please consider the ways that you can
```
Comment (andreakb) [inline _posts/2020-01-26-support-the-52Fund.md]:
```suggestion
Please read about the mission and consider the ways that you can
```

File: _posts/2020-01-26-support-the-52Fund.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+donate or support the cause.
+
+More on this can be learned by following the hash-tag
```
Comment (andreakb) [inline _posts/2020-01-26-support-the-52Fund.md]:
```suggestion
More on this issue can be learned by following the hash-tag
```

File: _posts/2020-01-26-support-the-52Fund.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+[#thatdarnedpetition][saa-9] on Twitter.
+
+*Disclaimer*: The author of this post is not a member of the SAA, their
```
Comment (andreakb) [inline _posts/2020-01-26-support-the-52Fund.md]:
```suggestion
*Disclaimer*: Though the author of this post is not a member of the SAA, their
```

File: _posts/2020-01-26-support-the-52Fund.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+> underrepresented identities in leadership positions, who advocate for success
+> of these communities and are willing to call out and address discriminatory
+> practices within the profession and in spaces supposedly designed to nurture
+> and support emerging leaders and change agents.
+
+We know you (the 52 signers) felt the nominees didn't bring enough "varied"
+experience, and felt they didn't bring multi-faceted connections to the
+concerns of the SAA's leadership (whatever that means?!).
+
+But what spoke to you and said:
```
Comment (andreakb) [inline _posts/2020-01-26-support-the-52Fund.md]:
```suggestion

```

File: _posts/2020-01-26-support-the-52Fund.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+* This truly didn't exist in the nominee's backgrounds, within SAA, or outside?
```
Comment (andreakb) [inline _posts/2020-01-26-support-the-52Fund.md]:
```suggestion
* The nominees didn't bring enough varied leadership experience from within and outside SAA
```

File: _posts/2020-01-26-support-the-52Fund.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+* That spoke to you and said, this couldn't be learned with a year in a VP
```
Comment (andreakb) [inline _posts/2020-01-26-support-the-52Fund.md]:
```suggestion
* The complexities of the role could not be learned in a year in a VP position with the mentor-ship of the current president
```

File: _posts/2020-01-26-support-the-52Fund.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+position with the mentorship of the current president?
```
Comment (andreakb) [inline _posts/2020-01-26-support-the-52Fund.md]:
```suggestion

```

File: _posts/2020-01-26-support-the-52Fund.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+* That you had the answer to what leadership should and does look like?
```
Comment (andreakb) [inline _posts/2020-01-26-support-the-52Fund.md]:
```suggestion
* Leadership should and does look a certain way that is not congruent with what the nominees offer.
```

File: _posts/2020-01-26-support-the-52Fund.md [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+* That you could speak better to the organization's needs than the existing
```
Comment (andreakb) [inline _posts/2020-01-26-support-the-52Fund.md]:
```suggestion
* A candidate not chosen by the elected nomination committee better serves SAA's needs
```

File: _posts/2020-01-26-support-the-52Fund.md [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+nominating committee?
```
Comment (andreakb) [inline _posts/2020-01-26-support-the-52Fund.md]:
```suggestion

```

File: _posts/2020-01-26-support-the-52Fund.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+
+The SAA community, and existing leadership (by virtue of _position_, or
+_**nature**_, have spoken and are looking to support Lae’l Hughes-Watkins
+[call][saa-7].
+
+They have setup a fund for underrepresented communities to access to have
+their voice on this situation as well and that is the *[#52Fund][saa-8]*.
+
+Please read about the mission, and please consider the ways that you can
+donate or support the cause.
```
Comment (andreakb) [inline _posts/2020-01-26-support-the-52Fund.md]:
```suggestion
participate, donate, or support the cause.
```

File: _posts/2020-01-26-support-the-52Fund.md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+demonstrate that. That is what we mean by leadership by nature. Processes and
+procedures can often help us but this time they are failing a group of good
+people.
+
+The SAA community, and existing leadership (by virtue of _position_, or
+_**nature**_, have spoken and are looking to support Lae’l Hughes-Watkins
+[call][saa-7].
+
+They have setup a fund for underrepresented communities to access to have
+their voice on this situation as well and that is the *[#52Fund][saa-8]*.
```
Comment (andreakb) [inline _posts/2020-01-26-support-the-52Fund.md]:
```suggestion

```

File: _posts/2020-01-26-support-the-52Fund.md [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+
+52 signers sought an end-around of the current Chair of the Nominating
+Committee. Themselves [seeking to][saa-6] do the following:
+
+> I think it will be critical to put a slate of candidates together that will
+> have a strong portfolio of success in making room for historically
+> underrepresented identities in leadership positions, who advocate for success
+> of these communities and are willing to call out and address discriminatory
+> practices within the profession and in spaces supposedly designed to nurture
+> and support emerging leaders and change agents.
```
Comment (andreakb) [inline _posts/2020-01-26-support-the-52Fund.md]: Add an attribution to this quote, like who exactly said it/wrote it and the context

**Comment by ross-spencer** [comment]
Thanks Andrea, hopefully the changes help. [Markdown rendering here](https://github.com/digital-preservation-is-people/digital-preservation-is-people.github.io/blob/30460fc2289b4d468d0a734bd1c65bae231a89d2/_posts/2020-01-26-support-the-52Fund.md).

**Comment by ross-spencer** [comment]
I might have addressed those AKB? Let me know: https://github.com/digital-preservation-is-people/digital-preservation-is-people.github.io/pull/21/commits/2c2e73349ba200e91d1242ff01251fb791d33eca

**Pull request merged**